designs/flow-manipulation-api/README.md

```diff
@@ -122,8 +122,9 @@ Based on [Design: Flow file format v2](https://github.com/node-red/node-red/wiki
 - static parseFlow(string or array)
   - create Flowset from flow.json string or parsed array
   - returns: FlowSet object
-- generator()
-  - iterate over all nodes in this flowset.
+- getAllNodesArray()
```

Comment:
`getNodes` would be a more consistent name for this method. Same comment across the other classes that include this function name.

Reply:
I agree.  I updated the PR for this.